inductiva/_cli/cmd_projects/download.py

```diff
@@ -0,0 +1,80 @@
+"""Download the project tasks files via CLI."""
+import concurrent.futures as cf
+import argparse
+
+import inductiva
+
+
+def download_projects(args):
+    """Download the project tasks files.
+
+    This function downloads the files of the tasks of a project.
+    Args:
+        args.project_name (str): Name of the project to download.
+        args.output_dir (str): Directory to save the downloaded files 
+            default(inductiva-output).
+        args.files (list): List of files to download.
+        args.std (bool): Flag to download the standard output and error files.
+    The downloads are done in parallel using a ThreadPoolExecutor with a maximum
+    of 10 workers.
+    """
+
+    project_name = args.project_name
+    output_dir = args.output_dir
+    files = args.files
+    std = args.std
+
+    if std:
+        files = ["stdout.txt", "stderr.txt"]
```

Comment:
Are the --std and --files exclusive? 

The std will overwrite the files if both arguments are used.. should we inform this to the user?

Reply:
Yes, you cant use both of them.